packages/wp-now/src/run-cli.ts

```diff
@@ -71,6 +77,40 @@ export async function runCli() {
 				}
 			}
 		)
+		.command(
+			'php <filePath>',
+			'Run the php command passing the arguments for php cli',
+			(yargs) => {
+				commonParameters(yargs);
+				yargs.positional('filePath', {
+					describe: 'Path to the PHP file to run',
+					type: 'string',
+				});
+			},
+			async (argv) => {
+				const spinner = startSpinner('Running the php command...');
+				try {
+					const options = {
+						projectPath: argv.path as string,
+						phpVersion: argv.php as SupportedPHPVersion,
+						wordPressVersion: argv.wp as string,
+					};
+					const result = await executePHPFile(
+						argv.filePath as string,
+						options
+					);
+					process.exit(result.status);
+				} catch (error) {
+					console.error(error);
```

Comment:
Recently we've merged the output wrapper. Could we use `output?.error()` here too?

Should we modify `shouldOutput` to return false when executed from the PHP CLI context?

Reply:
As long as this function is not a exported from the `wp-now` package, I think we're good here. There should be no way to trigger it from the outside. As for the internal use, printing too many errors sounds better than printing too few.

Reply:
> Should we modify shouldOutput to return false when executed from the PHP CLI context?

So no output when it's running in PHP CLI? When should it output, then?

Reply:
Ah you meant `wp-now php` for `shouldOutput=false`. Your call.

Reply:
Gotcha, that makes sense! FYI the Custom ESM Loaders thing won't be there in the built npm package. I'd love to get rid of it from the repo version as well but that would require a structural change that has other downsides.

Reply:
Thanks for confirming that. As far as I saw how npm-installed wp-now works, the nx build output is also gone there.

Reply:
I removed the wp-now logs but I'm keeping the error trace in case the execution fails.
https://github.com/WordPress/wordpress-playground/pull/336/commits/5157e9e3a26c29a64c810c5393b092e571bbbfa5